Volume/View Models/MagazineCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  FollowingMagazineCell.swift
+//  Volume
+//
+//  Created by Justin Ngai on 6/3/2022.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import SDWebImageSwiftUI
+import SwiftUI
+
+/// `MagazineCell` displays the images and name of a publication a user is currently following
+struct MagazineCell: View {
+
+    // TODO: Restore when backend setup
+    // let magazine: Magazine
+    // let largeFont: Bool
+
+    // TODO: Remove test values when backend setup
+    struct dummyMagazine {
```

Comment:
Do you plan to create a model file for these?

Reply:
Yeah there's a model file for Magazines but we haven't finalized the backend api yet.